runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -12173,7 +12173,6 @@ J9::Power::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&result
             return true;
             }
          break;
-
```

Comment:
May I ask you to drop this change from the pull request?  I'm you had a change here in an earlier version, but later removed it.